read-historical-data.md

```diff
@@ -152,3 +152,21 @@ Pay special attention to the following:
     > **Note:**
     >
     > You should use `@@` instead of `@` before `tidb_snapshot` because `@@` is used to denote the system variable while `@` is used to denote the user variable.
+
+## Restoring historical versions
```

Comment:
For task-oriented topics in our documentation, we usually use headings that start with the prototype of verbs OR "how to". In this case, it is recommended to use "Restore historical data" or "How to restore historical data".